src/useDownload.tsx

```diff
@@ -53,6 +54,8 @@ export function useDownload() {
       (tabs) => {
         const tabId = tabs[0].id || 0;
```

Comment:
At this point I think it might be smoother to store the tab object in state than select separate parts from the tab.